app/models/workers_scaling/experiment_resources_interface.rb

```diff
@@ -158,6 +141,38 @@ def get_worker_record_by_sm_uuid(sm_uuid)
     private
 
     ##
+    # Parses requested amount of workers to schedule to real amount.
```

Comment:
@mwrona z tym amount vs number - to co napisałem może być niegramatyczne przez mieszanie amout i number (Simulation Managery są raczej policzalne)